test/e2e_node/mirror_pod_test.go

```diff
@@ -81,8 +81,8 @@ var _ = framework.KubeDescribe("MirrorPod", func() {
 			ginkgo.By("check the mirror pod container image is updated")
 			pod, err = f.ClientSet.CoreV1().Pods(ns).Get(mirrorPodName, metav1.GetOptions{})
 			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
-			gomega.Expect(len(pod.Spec.Containers)).Should(gomega.Equal(1))
-			gomega.Expect(pod.Spec.Containers[0].Image).Should(gomega.Equal(image))
+			framework.ExpectEqual(pod.Spec.Containers, 1)
```

Comment:
We should not remove `len()`. That is the reason why the test job is failed now.

Reply:
Ooh, I didn’t noticed the failure, sorry for that.! Will fix it ASAP.